src/Signin.tsx

```diff
@@ -32,6 +30,7 @@ import {
 
 // Types
 import {RemoteDataState} from 'src/types'
+import {getMe} from './client'
```

Comment:
it's probably better to make this an absolute path, i.e. `src/client` rather than a relative one. It's more consistent with the rest of the imports in the file and the codebase and it helps keep this file more portable if it ever moves locations.